test/common/testinfra.py

```diff
@@ -581,6 +581,10 @@ def update_status(revision, context, last, changes):
                 status = statuses.get(context, None)
                 baseline = BASELINE_PRIORITY
 
+                # modify the baseline slightly to favor older pull requests, so that we don't
```

Comment:
In short: I prefer explicit, controlled criteria over undocumented API behavior.